passthrough.py

```diff
@@ -128,7 +131,8 @@ def fsync(self, path, fdatasync, fh):
 
 
 def main(mountpoint, root):
-    FUSE(Passthrough(root), mountpoint, nothreads=True, foreground=True)
+    FUSE(Passthrough(root), mountpoint, nothreads=True, foreground=True, **{'allow_other': True})
```

Comment:
Lol.  It's call new and green with Python ;)  Sorry about that I come from more of a Java/C# background and am still a newbie to Python.